hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -1,13 +1,16 @@
 import React from "react";
-
+import styles from "./TeamDetail.module.scss";
```

Comment:
Is this a relative import?

Reply:
it is but it's okay for the css file since we want the styling to be in the same directory as the react file